great_expectations/render/renderer/v3/suite_profile_notebook_renderer.py

```diff
@@ -213,6 +213,8 @@ def _add_rule_based_profiler_cells(self) -> None:
     name="{self._profiler_name}",
     expectation_suite_name=expectation_suite_name,
 )
+
+validator.expectation_suite = suite
```

Comment:
Per discussion with @alexsherstinsky, we need a way to associate our RBP result suite with our validator in the CLI workflow.

While we might want to refine this later on, the agreed-upon approach is to expose the suite attr through a setter so we can manually make that connection.